src/Runner.ts

```diff
@@ -118,6 +145,50 @@ function getPackages(dependencies: { [pkg: string]: string; }): string[] {
 }
 
 /**
+ * Generate an HTML page which represents the Runner properties
+ * @param param0 Properties from the Runner to be used to specify the document
```

Comment:
That is what the TypeScript services stubs out.  There is an [open issue](https://github.com/Microsoft/TypeScript/issues/11859) to actually support it, as I don't think TypeScript does at the moment.